misc/docker/README.md

```diff
@@ -27,7 +27,7 @@ store it in the misc/docker directory
  cp {logging,config}.yaml /etc/ansible-runner-service/.
  cp samples/* /usr/share/ansible-runner-service/
 ```
-*(the paths don't matter, they're just the ones I'm using for test)*
+*(the paths don't matter, they're just the ones I'm using for test. However, you do **need** artifacts, env, inventory and project within /usr/share/ansible-runner)*
```

Comment:
If we provide all the folders created, better remove this comment. In any case the comment would be:

(the paths don't matter, they're just the ones I'm using for test, if you change them just take that into account mapping the persistence folders in the command to start the container

Reply:
README updated in commit e97a9f9